frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx

```diff
@@ -63,14 +59,21 @@ export const RealDone = ({ options, title, label }: realDoneProps) => {
             <Checkbox checked={isAllSelected} />
             <ListItemText primary='All' />
           </MenuItem>
-          {options.map((data) => (
-            <MenuItem key={data} value={data}>
-              <Checkbox checked={selectedDoneColumn.includes(data)} />
-              <ListItemText primary={data} id={data} />
-            </MenuItem>
-          ))}
+          {doneColumns.map((column) => {
+            const isChecked = selectedDoneColumn.includes(column)
+            return (
+              <MenuItem key={column} value={column}>
+                <Checkbox checked={isChecked} />
+                <ListItemText primary={column} />
+              </MenuItem>
+            )
+          })}
         </Select>
-        <FormHelperText>{isEmptyRealDoneData ? errorHelperText() : DEFAULT_HELPER_TEXT}</FormHelperText>
+        {isEmptyRealDoneData && (
```

Comment:
set default helper text here，ref to main branch

Reply:
got that, will fix that soon